tinyms/serving/server/server.py

```diff
@@ -78,8 +79,16 @@ class _FlaskServer(object):
     in subprocess.
     """
 
-    def __init__(self):
+    def __init__(self, host='127.0.0.1', port=5000, server_path=None):
```

Comment:
I don't think jump the server_path to `temp.json` is a good idea, could u try to set environment variable like `SERVING_PATH`？

Reply:
I think it’s not a good way to configure a default path by changing environment variables. The python program has no way to set an environment variable that other processes can access. Violent modification of environment variables through the command line may cause the user’s original environment variables to be changed. modify. A temporary file is more appropriate in my opinion